pkg/api/componentreadiness/component_report.go

```diff
@@ -126,6 +127,10 @@ func GetComponentReportFromBigQuery(
 		variantJunitTableOverrides: variantJunitTableOverrides,
 	}
 
+	if os.Getenv("DEV_MODE") == "1" {
+		return generator.GenerateReport(ctx)
+	}
```

Comment:
This double caching, the report as well as the underlying bigquery queries, is a real pain for devel. I'd almost like to remove it and let CR recalc the report every time, but at least for development it's very helpful to benefit from bigquery caching, but not for report generation.

Reply:
IIRC we looked at removing the report caching but there is still a significant hit (in ui loading terms anyway) that made it worthwhile to keep it.

Reply:
I'm showing about 1.5s request response times with this setup on a very fast laptop, so there's definitely some significant computation being done and could be slower still in prod.